data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg

```diff
@@ -699,15 +700,24 @@ Either way, nothing will be gained by your intrusion into our woods. Now advance
     [event]
         name=time over
 
-        [message]
-            speaker=Muff Toras
-            message= _ "My reinforcements are here!"
-        [/message]
-
-        [message]
-            speaker=Arvith
-            message= _ "He escaped us..."
-        [/message]
+        [if]
+            [variable]
+                name=found_kidnappers
+                boolean_equals=no
+            [/variable]
+            [then]
+                [message]
+                    speaker=Arvith
+                    message= _ "These elves have delayed us too long, the kidnappers’ trail will have faded by now."
+                [/message]
+            [/then]
+            [else]
+                [message]
+                    speaker=Arvith
+                    message= _ "More elves are arriving; even if they believe me, the kidnappers’ trail is sure to fade before we can follow."
```

Comment:
```suggestion
                    message= _ "More elves are arriving! The kidnappers’ trail is sure to fade before we can follow."
```

Reply:
Thanks. I'm going to leave the "even if they believe me" in, because it doesn't imply that Arvith could defeat the elvish reinforcements. I'm assuming a few scouts would slow down Arvith, and a large number of reinforcements would follow.

I guess the dark adept (who gets released alive at the start of S03) finds out that there's an army of elves following Arvith and waiting for the Council's decision; but that's not our protagonists' problem.